### PR TITLE
OEL-759: Add facts and figure and description list paragraphs to the example page.

### PR DESCRIPTION
**Pull request by escuriola**
### Description

We need to:
* Enable paragraphs Facts and figures and Description list at Body field in OE Showcase Page content type.
* Provide of example of those two paragraphs at the Demo Page.
* Provide tests for those two paragraphs at the PageTest.php

### CODE REVIEW

**Review by drishu — DISMISSED** [review]

**Review by abel-santos-corral — DISMISSED** [review]

**Review by abel-santos-corral — CHANGES_REQUESTED** [review]
Please, activate the paragraphs Facts and figures (not Fact sub-paragraph) at Body of Demo Page content type.

Many thanks in advance.

**Review by abel-santos-corral — DISMISSED** [review]

File: modules/oe_showcase_page/oe_showcase_page.info.yml [hunk 1]
```diff
@@ -11,12 +11,14 @@ dependencies:
...
   - oe_paragraphs:oe_paragraphs_banner
   - oe_paragraphs:oe_paragraphs_chart
+  - oe_paragraphs:oe_paragraphs_description_list
   - oe_paragraphs:oe_paragraphs_document
   - oe_paragraphs:oe_paragraphs_map
   - oe_paragraphs:oe_paragraphs_media
   - oe_paragraphs:oe_paragraphs_social_feed
   - oe_paragraphs:oe_paragraphs_timeline
   - openeuropa:oe_webtools
+  - openeuropa:oe_bootstrap_theme_paragraphs
```
Comment (drishu) [inline modules/oe_showcase_page/oe_showcase_page.info.yml]: attention, this will soon be oe_whitelabel_paragraphs

**Review by drishu — CHANGES_REQUESTED** [review]
the paragraphs will move so this needs to be re-tested after, I also dont extra lines in the test as the ticket requested

**Review by drishu — DISMISSED** [review]

File: modules/oe_showcase_page/oe_showcase_page.info.yml (hunk 1 above)
Comment (abel-santos-corral) [reply modules/oe_showcase_page/oe_showcase_page.info.yml:21]: We had this change by rebasing 1.x.

**Review by brummbar — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
Blind approval as requested by @brummbar

**Pull request merged**